### PR TITLE
Shipping Address

### PR DESCRIPTION
**Pull request by sagemlee**
Add user's shipping address to mutations and resolvers, test update of shipping address, all tests passing

## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- Mutations are able to update email and description but not shipping address

## What is the new behavior?

-You can now update shipping address with a mutation

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: code/api/src/modules/user/model.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = function(sequelize, DataTypes) {
       type: DataTypes.STRING
     },
     shippingAddress: {
-      type: DataTypes.DATEONLY
+      type: DataTypes.STRING
```
Comment (sagemlee) [inline code/api/src/modules/user/model.js:22]: This was a typo from a previous merge, this is what it should have been.

File: code/web/src/modules/user/api/actions.js [hunk 2]
```diff
@@ -27,7 +27,37 @@ export function setUser(token, user) {
 
 // Login a user using credentials
 export function login(userCredentials, isLoading = true) {
-    return (dispatch) => {
+  return dispatch => {
```
Comment (sagemlee) [inline code/web/src/modules/user/api/actions.js:30]: Not sure why this is showing up like this, I believe this is what we already have in main

Comment (edignot) [reply code/web/src/modules/user/api/actions.js:30]: This is the same action that I was prompted to solve conflicts. It should be here anyways

**Comment by edignot** [comment]
so as I understand userUpdate mutation type is ready to use?

**Comment by sagemlee** [comment]
Everything should be "update-able" aside from the imagine, not sure if you decided to do that in a separate request

**Comment by KyleWong2510** [comment]
Great job getting our shippingAddress set up!

File: code/api/src/modules/user/mutations.test.js [hunk 3]
```diff
@@ -4,8 +4,7 @@ import graphqlHTTP from 'express-graphql'
 import schema from '../../setup/schema'
 import models from '../../setup/models'
 import connection from '../../setup/database'
-
-const params = require('../../config/params.json');
+import params from'../../config/params.json'
```
Comment (arpariseau) [inline code/api/src/modules/user/mutations.test.js:7]: Just copied it from the seeders, so I guess import makes it look a lot cleaner

File: code/api/src/modules/user/mutations.test.js [hunk 4]
```diff
@@ -74,12 +73,24 @@ describe('user mutations', () => {
...
+  it('changes shippingAddress', async () => {
+    const response = await request(server)
+      .post('/')
+      .send({
+        query:
+          'mutation { userUpdate(id: 2, email: "user@crate.com", shippingAddress: "main.st") { shippingAddress } }'
+      })
+      .expect(200)
+
+    const test = await models.User.findByPk(2)
```
Comment (arpariseau) [inline code/api/src/modules/user/mutations.test.js:85]: 👍

**Pull request merged**